### DISCUSSION TITLE
mikro orm + nuxt 3 nitro server

### DISCUSSION THREAD

**Discussion by max5432112345** [mikro-orm/mikro-orm#3255] [Q&A]
**Describe the bug**
How to work with that if no nex() function
```
import { RequestContext } from '@mikro-orm/core'

import { DI } from '../data-source'

export default defineEventHandler(async (event) => {
  RequestContext.create(DI.orm.em, () => {
    setTimeout(() => {}, 200)
  })
})
```
Legacy support is possible using https://github.com/unjs/h3, but it advised to avoid legacy handlers as much as you can.
```
export default (req, res, next) => {
  console.log('Legacy middleware')
  next()
}
```
But that also not work

**Stack trace**
```
[nitro] [dev] [unhandledRejection] ValidationError: Using global EntityManager instance methods for context specific actions is disallowed. If you need to work with the global instance's iden
tity map, use `allowGlobalContext` configuration option or `fork()` instead.
```

**To Reproduce**
https://github.com/nuxt/framework/issues/5603

**Versions**

- Operating System: `Linux`
- Node Version:     `v16.15.1`
- Nuxt Version:     `3.0.0-rc.4`
- Package Manager:  `npm@8.12.2`
- Typescript: `Version 4.7.4`
- Mikro-orm: `5.2.1`
- Driver: `mariadb`

**Comment by B4nan** [comment]
This is out of scope, can't help with integrations to frameworks I never used, definitely not if you dont provide complete (but minimal) reproduction. 

Note that you need middleware, not event handler. There needs to be some closure (= some next function).
